app.js

```diff
@@ -0,0 +1,57 @@
+"use strict";
+
+const path = require('path');
+
+const bodyParser = require('body-parser');
+const express = require('express');
+
+const config = require('./app/config/').Config;
```

Comment:
yes I know, but the only argument AFAIK to add a index-file like this, is to shorten the import path, or to make it easier to refactor later. Would rather be using the whole path instead of having to re-export everything in a index-file